### ISSUE TITLE
Implement SA1132 (DoNotCombineFields)

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#825]
**Category:** Readability
**Title:** Do not combine field definitions
**Description:** C# allows more than one field to be defined in a single field declaration syntax. When fields are combined, it becomes difficult to impossible to properly document them, and difficult to identify each field defined in a type.

**Example:**

The following code shows an example which would produce this warning:

```csharp
int x, y;
```

To resolve the violation, define in field separately:

```csharp
int x;
int y;
```

**Comment by vweijsters** [comment]
:+1: Big +1 for me, that's one feature they should have never introduced IMHO.

**Comment by johnkoerner** [comment]
I agree with this one 100%.
